boards/shields/adafruit_2_8_tft_touch_v2/adafruit_2_8_tft_touch_v2.overlay

```diff
@@ -17,3 +17,13 @@
 		cmd-data-gpios = <&arduino_header 15 GPIO_ACTIVE_LOW>;	/* D9 */
 	};
 };
+
+&arduino_i2c {
+	touch_controller: ft5336@38 {
+		compatible = "focaltech,ft5336";
+		reg = <0x38>;
+		label = "FT5336";
+		/* Uncomment if IRQ line is available (requires to solder jumper) */
```

Comment:
wondering if there's any way to conditionally do this in DTS via Kconfig, rather than requiring source tree modification? @galak

Reply:
This can be solved in the driver code, KSCAN_FT5336_INTERRUPT could be removed and DT_NODE_EXISTS(...) used instead.

Reply:
@gmarull Please consider to remove KSCAN_FT5336_INTERRUPT and use DT_NODE_EXISTS(...) here or in a subsequent PR.

Reply:
@jfischer-phytec-iot Thanks for the suggestion. I'll try to improve using this macro in another PR as I also want to add the overlay for the test board I'm using.